### PR TITLE
Add server logging for failure responses

### PR DESCRIPTION
**Pull request by seanf**

### CODE REVIEW

**Comment by huangp** [comment]
:+1:

File: zanata-war/src/main/java/org/zanata/rest/editor/service/TranslationService.java [hunk 1]
```diff
@@ -80,7 +82,10 @@ public Response get(String localeId, String ids) {
         }
         List<Long> idList = TransUnitUtils.filterAndConvertIdsToList(ids);
         if (idList.size() > TransUnitUtils.MAX_SIZE) {
-            return Response.status(Response.Status.FORBIDDEN).build();
+            String msg = String.format("More than %d results.", TransUnitUtils.MAX_SIZE);
+            log.warn(msg);
+            return Response.status(Response.Status.FORBIDDEN).entity(msg)
```
Comment (djansen-redhat) [inline zanata-war/src/main/java/org/zanata/rest/editor/service/TranslationService.java]: Why did this line wrap, but not the one 2 lines above it? Weird.

**Comment by djansen-redhat** [comment]
Any hints on how to trigger these?

**Comment by seanf** [comment]
> Any hints on how to trigger these?

You might be able to use a custom client which sent an invalid request (too many transunit IDs), or perhaps send an illegal document for translation, but otherwise it would require a unit test with mock objects or something like Byteman which can inject exceptions: http://byteman.jboss.org/.  All of which is probably overkill for such a simple change.

**Comment by djansen-redhat** [comment]
Agreed.

**Pull request merged**